test/sandbox/app/controllers/integration_examples_controller.rb

```diff
@@ -3,6 +3,8 @@
 class IntegrationExamplesController < ActionController::Base
   layout false
 
+  helper_method :current_user
```

Comment:
I assumed the default test controller was an ok place to define this, and I didn't want to obfuscate the intention of the test by creating a specific controller.